src/components/views/SettingsView/index.tsx

```diff
@@ -95,6 +125,7 @@ const SettingsView = () => {
       }),
     ],
     [
+      deviceTheme,
       isAppleAuthorized,
```

Comment:
From my local testing, it looks like we also need to add `setDeviceTheme` to this list

![Screen Shot 2021-06-25 at 12 58 01 PM](https://user-images.githubusercontent.com/21055469/123478909-fe2b1d00-d5b4-11eb-90cb-871e5c80b1c6.png)